src/components/AppNavigation/Settings/SettingsAddressbook.vue

```diff
@@ -209,6 +217,25 @@ export default {
 		addressbookUrl() {
 			return window.location.origin + this.addressbook.url
 		},
+		
+		contacts() {
+		return Object.values(this.addressbook.contacts);
+		},
+
+		groups() {
+		const allGroups = this.contacts
+			.flatMap(contact => contact.vCard.getAllProperties('categories').map(prop => prop.getFirstValue()));
+		// Deduplicate
+		return [...new Set(allGroups)];
+		},
+
+		contactsCount() {
+      	return this.contacts.length;
```

Comment:
indentation looks off now. are tabs and spaces mixed up?

Reply:
The changes of 6851011 were disregarded when resetting. I made the reset again by committing them.